login.html

```diff
@@ -0,0 +1,59 @@
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href=https://breezeui.netlify.app/styles/breezeUI.css>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="stylesheet" href="main.css">
+    <link rel="icon" href="assets/images/paws-and-beyond.png" type="image/png">
+    <title>paws & beyond | Login</title>
+</head>
+<body>
+    <header class="flex-space-btwn header-container">
+        <div class="flex-center">
+            <img class="p-b-logo" src="assets/images/paws-and-beyond-logo.png" 
+                alt="paws and beyond logo">
+            <span class="p-lg">paws<span class="txt-accent">&</span>beyond</span><span class="txt-accent txt-bold p-lg">.</span>
+        </div>
+        <div class="flex-center flex-space-btwn search-bar">
+            <input class="search-input p-sm" 
+                placeholder="Let's find what you are looking for...">
+            <i class="fa fa-search search-icon"></i>
+        </div>
+        <nav class="flex-layout">
+            <a class="nav-pill" href="index.html"><img class="nav-pill-img" src="assets/images/home.png"></a>
+            <a class="nav-pill" href="wishlist.html"><img class="nav-pill-img" src="assets/images/heart.png"></a>
+            <a class="nav-pill" href="cart.html"><img class="nav-pill-img" src="assets/images/bag.png"></a>
+            <a class="nav-pill active-nav-pill" href="login.html"><img class="nav-pill-img" src="assets/images/user.png"></a>
+        </nav>
+    </header>
+
+    <div class="flex-space-btwn flex-center space-around login-parent">
+        <div class="flex-column">
+            <img class="auth-landing-img" src="assets/images/auth-background.png">
+            <a href="index.html" class="checkout-btn link-btn browse-btn">Browse Products <i class="fa fa-chevron-circle-right mg-rl-5px"></i></a>
+        </div>
+        <div>
+            <form class="flex-column glass-card login-form">
+                <div class="flex-column flex-center">
+                    <span class="p-md form-heading">Login</span>
+                    <input class="form-control mg-tb-1 form-input" type="text" placeholder="Email"/>
+                    <input class="form-control input-lg mg-tb-1 form-input" type="password" placeholder="Password"/>
+                    <div class="flex-layout flex-space-btwn width-100">
+                        <div class="flex-layout">
+                            <input type="checkbox"> 
+                            <label class="fs-13px">Remember me</label>
+                        </div>
+                        <a class="link-txt" href="">Forgot Password ?</a>    
+                    </div>
+                    <button class="checkout-btn">Login</button>
+                    <a class="link-txt" href="signup.html">Create new account >></a>
```

Comment:
you can add an arrow icon for better Visual.
Very nice work go forward🚀

Reply:
Sure will do that. Thank you 😄